extensions/resteasy-reactive/rest-client-reactive/deployment/src/test/java/io/quarkus/rest/client/reactive/headers/ClientHeaderParamFromPropertyTest.java

```diff
@@ -78,5 +93,11 @@ public interface Client {
         @GET
         @ClientHeaderParam(name = "some-other-header", value = "${non-existent-property}", required = false)
         String missingNonRequiredProperty();
+
+        @GET
+        @ClientHeaderParam(name = "additional-text", value = "Bearer ${my.property-value}")
```

Comment:
How about we add some text after `}` as well?

Reply:
Good idea, done